src/sentry/tasks/clear_expired_resolutions.py

```diff
@@ -16,7 +16,7 @@ def clear_expired_resolutions(release_id):
     This is currently only used for ``in_next_release`` resolutions.
     """
     try:
-        release = Release.objects.get_from_cache(id=release_id)
+        release = Release.objects.get(id=release_id)
```

Comment:
Isn't `release_id` a unique identifier?

Reply:
It is not marked as cached, so it never ended up in Redis and this call always went to db. I just try to make the existing code more explicit, I don't really want to change the behavior because even if `release_id` is pk (probably) I don't know if this impacts consistency guarantees.